### PR TITLE
{devel}[GCCcore/6.4.0] PCRE v8.42

### PR DESCRIPTION
**Pull request by JackPerdue**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/p/PCRE/PCRE-8.42-GCCcore-6.4.0.eb [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+description = """
+ The PCRE library is a set of functions that implement regular expression
+ pattern matching using the same syntax and semantics as Perl 5.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['ftp://ftp.csx.cam.ac.uk/pub/software/programming/pcre/']
```
Comment (boegel) [inline easybuild/easyconfigs/p/PCRE/PCRE-8.42-GCCcore-6.4.0.eb]: @JackPerdue Where possible, we prefer `http://` over `ftp://`; in this case, we could include both, but have `http://` first:

```python
source_urls = [
    'https://ftp.pcre.org/pub/pcre/',
    'ftp://ftp.csx.cam.ac.uk/pub/software/programming/pcre/',
]

**Review by boegel — DISMISSED** [review]

**Comment by migueldiascosta** [comment]
@JackPerdue, this PR is being closed for the following reason(s): closing and reopening to trigger tests.

File: easybuild/easyconfigs/p/PCRE/PCRE-8.42-GCCcore-6.4.0.eb (hunk 1 above)
Comment (akesandgren) [reply easybuild/easyconfigs/p/PCRE/PCRE-8.42-GCCcore-6.4.0.eb]: Fixed

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
b-an04.hpc2n.umu.se - Linux ubuntu 16.04, x86_64, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/52bc9b13a1b4b0e55c33016c62314300 for a full test report.

**Review by akesandgren — APPROVED** [review]
LGTM

**Comment by akesandgren** [comment]
Going in, thanks @JackPerdue!

**Pull request merged**